source/chapters/user_interface.rst

```diff
@@ -237,10 +237,11 @@ then toggling the :menuselection:`Skin Settings --> Parallel Waveforms` option.
   The line on the vinyl widget rotates if the track moves. It is similar to the
   position marker found on scratch records. Use the mouse on the vinyl widget to
   pause, scratch, spin-back or throw tracks - just like a real record.
-  When performing :ref:`Loop rolls <interface-looping>` or right-clicking on the
```

Comment:
The term "Loop rolls" is not used in the linked chapter. Please adjust unify the terms.

Reply:
I changed it to "rolling loops", that's what we use in Interface -> Looping.